portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,10 +45,33 @@ public void init() {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    /*
     //Function returns all comments as json string
     String commentsJsonFinal = commentsJson + "}";
     response.setContentType("application/json;");
     response.getWriter().println(commentsJsonFinal);
+    */
```

Comment:
what do we have this commented function here?

Reply:
that is the chunk of code used for displaying comments prior to implementing the DataStore. I kept it in as I can use it as a backup until I fully integrate the load and store implementations.

Reply:
sg, just add a TODO comment to clean it up later.